### ISSUE TITLE
dashboard: Display accurate robot footprint

### ISSUE THREAD

**Issue by aaronchongth** [open-rmf/rmf-web#499]
### Status

Blocked by open-rmf/rmf_ros2#121.

### Description

In order to more accurately display the footprint of the robot on the dashboard, we can render the footprint based on the size provided by its current trajectory.

### Implementation considerations

Robot footprints are only available through the trajectories provided by the Trajectory Server, hence the trajectories thus far has been accurately portrayed. Since the introduction of `ResponsiveWait`, even when the robot is idle, the footprint should be readily available to be used to render the footprint.

**Comment by koonpeng** [comment]
Opened another issue to address this open-rmf/rmf_ros2#121. Even with `ResponsiveWait` there are still some architectural problems that should be fixed at the source instead of introducing needlessly complex synchronisations between multiple servers.

**Comment by aaronchongth** [comment]
This is resolved using the icon scale
